hnn_core/network.py

```diff
@@ -375,6 +372,7 @@ def __init__(self, params, add_drives_from_params=False, legacy_mode=True):
 
         # contents of pos_dict determines all downstream inferences of
         # cell counts, real and artificial
+        self.n_cells = 0
```

Comment:
`n_cells` is not documented. Revealed because you moved the declaration to the init :)

also I think user should not be allowed to set it, so make it private. If you want to give them access to read this attribute, use `@property` decorator

Reply:
Oh good catch. I made `n_cells` private in https://github.com/jonescompneurolab/hnn-core/pull/436/commits/96d7c637ec2184325d036392962d239d90b809dd and don't see any reason why users should be accessing it as it's primarily just for testing.